lib/config/options/index.ts

```diff
@@ -1855,6 +1855,14 @@ const options: RenovateOptions[] = [
     default: null,
     cli: false,
   },
+  {
+    name: 'prTitleStrict',
+    description:
+      'Whether to bypass appending extra context to the Pull Request title.',
+    type: 'boolean',
```

Comment:
```suggestion
    type: 'boolean',
    experimental: true,
```

Reply:
Maybe let's mark it as experimental and move forward?

Reply:
Sounds good to me - thanks @rarkins